cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -60,8 +63,11 @@ NODE_PFS=($(move_sriov_pfs_netns_to_node "$SRIOV_NODE" "$PF_COUNT_PER_NODE"))
 SRIOV_NODE_CMD="docker exec -it -d ${SRIOV_NODE}"
 ${SRIOV_NODE_CMD} mount -o remount,rw /sys     # kind remounts it as readonly when it starts, we need it to be writeable
 ${SRIOV_NODE_CMD} chmod 666 /dev/vfio/vfio
-_kubectl label node $SRIOV_NODE sriov=true
+_kubectl label node $SRIOV_NODE $SRIOV_NODE_LABEL
 
 for pf in "${NODE_PFS[@]}"; do
   docker exec $SRIOV_NODE bash -c "echo 0 > /sys/class/net/$pf/device/sriov_numvfs"
 done
+
+used_pfs_yaml_format_array=$(sed 's/ /, /g' <<< "[\"${NODE_PFS[@]}\"]")
```

Comment:
I found this neat join implementation: `function join_by { local IFS="$1"; shift; echo "$*"; }` , then you call it with `join_by , [\"${foolist[*]}\"]`. (the only difference is that there is no space after the `,`.

Saw it [here](https://stackoverflow.com/questions/1527049/how-can-i-join-elements-of-an-array-in-bash) on the accepted answer (it includes a more generalize solution if you need multi-char separators).

You may probably ask why it is better than `sed`, right? :)
Well, it is using bash built-in, so it does not execute it in a sub-shell.